test/unit/metrics/test_generated_metrics.py

```diff
@@ -7,8 +7,12 @@
 
 # ======================================================
 
+apx0 = 1e-15
+apx1 = 1 - apx0
```

Comment:
Should this be handled through `pytest.approx()` ?

Reply:
This is not used in assertions, this is used in the construction of an alternate y_pred. The goal is to avoid taking sklearn.log_loss([0], [1]) (which would mathematically result in +infinity, but sklearn has a default behavior that just changed between two versions).

Reply:
Could you add a comment about that?

Reply:
done!